src/Transports/MassTransit.KafkaIntegration/KafkaIntegration/Serializers/DefaultSerializationFactory.cs

```diff
@@ -0,0 +1,23 @@
+namespace MassTransit.KafkaIntegration.Serializers
+{
+    using System.Net.Mime;
+    using Confluent.Kafka;
+    using Serialization;
+
+
+    public class DefaultKafkaSerializerFactory :
+        IKafkaSerializerFactory
+    {
+        public ContentType ContentType => SystemTextJsonMessageSerializer.JsonContentType;
+
+        public IDeserializer<T> GetDeserializer<T>()
+        {
+            return DeserializerTypes.TryGet<T>() ?? new MassTransitJsonDeserializer<T>();
```

Comment:
What is this, a global variable? Ugh.

Reply:
these are just defaults to replica confluent Consumer logic: https://github.com/MassTransit/MassTransit/blob/9d2b72c67d06ae1ef2c01282bf8361b40b7f74b5/src/Transports/MassTransit.KafkaIntegration/KafkaIntegration/Serializers/DeserializerTypes.cs#L10